core/state/statedb.go

```diff
@@ -286,6 +286,24 @@ func (s *StateDB) GetBalance(addr common.Address) *big.Int {
 	return common.Big0
 }
 
+func (s *StateDB) GetNonceLittleEndian(address common.Address) []byte {
```

Comment:
it's weird that you need this, the data in the db should be stored directly in little endian format

Reply:
You're right.

Reply:
Is this the case on a big-endian system?  Does Geth even care about supporting big-endian systems?

Reply:
Yes, big-endian systems should be supported, big-endian is still king in the high-end server space and that's where geth might eventually find its niche if the current trend continues.

Reply:
okay, we've agreed on the call that the code is correct as it currently is and no change is needed.